larndsim/quenching.py

```diff
@@ -15,7 +15,7 @@
 logger.info("QUENCHING MODULE PARAMETERS")
 
 @cuda.jit
-def quench(tracks, mode):
+def quench(tracks, light_dat, mode):
```

Comment:
Please add to the docstring a description of `light_dat`

Reply:
fixed in https://github.com/DUNE/larnd-sim/pull/39/commits/2b23cdbe8b76f7f729a6865e5329eff8bbf2600c